pkg/sync/kubernetes/kubernetes_sync.go

```diff
@@ -65,11 +65,13 @@ func (k *Sync) Init(ctx context.Context) error {
 		return err
 	}
 
-	k.readClient, err = client.New(clusterConfig, client.Options{Scheme: scheme.Scheme})
+	readClient, err := client.New(clusterConfig, client.Options{Scheme: scheme.Scheme})
```

Comment:
Minor - the rest of the work looks good. But any reason for this change :) ?

Reply:
not really a big reason when I think about it now, reverting :)

Reply:
Thanks, just wanted to see if this is a common pattern or a best practice :)